web/styles/zulip.css

```diff
@@ -2410,3 +2410,7 @@ body:not(.hide-left-sidebar) {
         border: 1px solid var(--color-border-personal-menu-avatar);
     }
 }
+
+.strikethrough {
+    text-decoration: line-through;
+}
```

Comment:
Not sure if having classes like this is okay or not according to current practices.

Reply:
I think the class name is fine. Other option could be `soft-removed-user`. Might be better to scope the CSS rule only for the subscribers though and move this to `subscriptions.css`.

Reply:
Fixed!